tests/appsec/api_security/test_schemas.py

```diff
@@ -25,9 +25,9 @@ def equal_without_meta(t1, t2):
 def equal_value(t1, t2):
     """compare two schema type values, ignoring any metadata"""
     if isinstance(t1, list) and isinstance(t2, list):
-        return len(t1) == len(t2) and all(equal_without_meta(a, b) for a, b in zip(t1, t2))
```

Comment:
I removed the check for length because some frameworks might be adding specific parameters that could cause this check to fail.